<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorClass</class>
 <widget class="QMainWindow" name="ItemEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="widget_mainPage">
     <widget class="QLabel" name="label_mainPage_title">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Goudy Stout</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ltem Editor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mainPage_newItem">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>200</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>New Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_mainPage_exit">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>250</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_createItem">
     <widget class="QListView" name="list_createItem_itemType">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_createItem_name">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_createItem_name">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="input_createItem_enhancementBonus">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>120</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="list_createItem_enhancement">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_createItem_itemType">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Item Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_createItem_enhancement">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enhancement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_createItem_enhancementBonus">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>100</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bonus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_createItem_createAndSave">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>320</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Create and Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_createItem_exit">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>320</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
